src/hotspot/share/prims/jvmti.xml

```diff
@@ -1915,8 +1917,12 @@ jvmtiEnv *jvmti;
         </param>
       </parameters>
       <errors>
-        <error id="JVMTI_ERROR_UNSUPPORTED_OPERATION">
-          <paramlink id="thread"/> is a virtual thread.
+        <error id="JVMTI_ERROR_THREAD_NOT_SUSPENDED">
+          Thread is a virtual thread and was not suspended and was not the current thread.
+        </error>
+        <error id="JVMTI_ERROR_OPAQUE_FRAME">
+          The thread is a suspended virtual thread and the implementation was unable
+          to throw an asynchronous exception from this frame.
```

Comment:
This part no longer has wording similar to the general description of JVMTI_ERROR_OPAQUE_FRAME below. Maybe that was understood and intended when the rewording was done. Just want to make sure you are aware of it.

Reply:
What part of the statement does not match?
Should we say "from the current frame" instead of "from this frame"?

Reply:
They are both trying to convey the same thing, but using completely different wording to do so.  One says "the implementation was unable to throw an asynchronous exception", and the other says "the function cannot be performed". One says "from this frame", and the other says "on the thread's current frame". The meaning is the same, but the wording should be consistent.

Reply:
I feel that it is a feature in the spec to say differently in specific case vs common case.
It should help to understand each case better.
In this particular case, it can be useful to align wording with "the current frame".

Reply:
I can see that reasoning for "unable to throw an asynchronous exception" and "cannot be performed", but what about "the implementation" vs "the function". Can't they both be the same?

Reply:
I was thinking about the same.
If I remember correctly, one of the Alan's suggestions was like this.
The problem is the spec has several variations for it:
 - function, operation, implementation...
It is hard or impossible to make this completely consistent.
But I have a doubt it is very important.
The spec might be boring to read if it is fully consistent. :)

Reply:
I've pushed an update with the change:
  `from this frame` => f`rom the current frame`

Also, updated the CSR.